tracing-subscriber/src/filter/env/field.rs

```diff
@@ -55,12 +68,113 @@ pub(crate) enum ValueMatch {
     Pat(Box<MatchPattern>),
 }
 
+// === impl ValueMatch ===
```

Comment:
Preferably I would have added this block just above the `==== impl ValueMatchInternal ====` block but that would have introduced many more line changes as some existing impl where outside of their section.